tests/translate/translate_remap_profile_2d.py

```diff
@@ -50,6 +49,7 @@ def make_storage_data_input_vars(self, inputs, storage_vars=None):
                 del inputs[serialname]
 
     def compute(self, inputs):
+        self.compute_func = profile.RemapProfile(inputs["kord"], inputs["iv"])
```

Comment:
ah yes, when it depends on input data arguments, we can't initialize in the init.